data/json/monstergroups/wilderness.json

```diff
@@ -3,22 +3,23 @@
     "type": "monstergroup",
     "name": "GROUP_CAVE",
     "default": "mon_null",
-    "//": "Current count is 925.  Note that 'freq' units are tenth of a percent, with default filling in the gap.",
+    "//": "Current count is 975.  Note that 'freq' units are tenth of a percent, with default filling in the gap.",
```

Comment:
This actually should have gone down, not up, with the reduction of mon_bat more than counterbalancing for the addition of mon_stag_beetle_larva; double-check, please.

Reply:
This went up after recalculation, because the last commit that added to this group (the mole cricket) did not change this line.